balloc/balloc.c

```diff
@@ -301,7 +301,7 @@ M0_INTERNAL void m0_balloc_debug_dump_group_extent(const char *tag,
 if (ENABLE_BALLOC_DUMP) {
 	struct m0_lext	*ex;
 
-	if (grp == NULL || grp->bgi_extents == NULL)
+	if (grp == NULL || grp->bgi_extents_loaded)
```

Comment:
guess this has to be grp->bgi_extents_loaded == false :)

Reply:
good catch, Vidyadhar. 
Updated, Thank you!